pkg/asn1/testsm/test_sm.proto

```diff
@@ -50,81 +53,87 @@ enum TestEnumeratedExtensible {
     TEST_ENUMERATED_EXTENSIBLE_ENUM6 = 5;
 };
 
+//{TEST-EnumeratedExtensible}
+enum TestFullyOptionalSequenceItem4 {
+    TEST_FULLY_OPTIONAL_SEQUENCE_ITEM4_ONE = 0;
+    TEST_FULLY_OPTIONAL_SEQUENCE_ITEM4_TWO = 1;
+};
+
 // sequence from tes_sm.asn1:15
 // {TEST-UnconstrainedInt}
 message TestUnconstrainedInt {
-    int32 attr_uci_a = 1 [ json_name="attrUciA"];
-    int32 attr_uci_b = 2 [ json_name="attrUciB"];
+    int32 attr_uci_a = 1 [json_name = "attrUciA"];
+    int32 attr_uci_b = 2 [json_name = "attrUciB"];
 };
 
 // sequence from tes_sm.asn1:20
 // {TEST-ConstrainedInt}
 message TestConstrainedInt {
     // @inject_tag: aper:"valueLB:10,valueUB:100"
-    int32 attr_ci_a = 1 [ json_name="attrCiA"];
-    // @inject_tag: aper:"valueLB:10"
-    int32 attr_ci_b = 2 [ json_name="attrCiB"];
+    int32 attr_ci_a = 1 [json_name = "attrCiA"];
+    // @inject_tag: aper:"valueLB:10,valueUB:4294967295"
```

Comment:
yes - why not have a long value - i think you should have a medium too - 65535 or smaller